opensquirrel/writer.py

```diff
@@ -17,6 +17,10 @@ def visit_int(self, i: Int):
     def visit_float(self, f: Float):
         return f"{f.value:.{self.number_of_significant_digits}}"
 
+    def visit_measure(self, measurement: Measure):
+        formatted_args = (arg.accept(self) for arg in measurement.arguments)
+        self.output += f"{measurement.name} {', '.join(formatted_args)}\n"
```

Comment:
A measure operation can only have __one__ argument.
```suggestion
        self.output += f"{measurement.name} {measurement.arguments[0].accept(self)}\n"
```